packages/web/src/components/TeeResults.vue

```diff
@@ -156,16 +218,71 @@ interface Props {
   trackForm?: any,
   tracksResults: TrackChoice[] | any[],
   trackElement: any;
-  debug?: boolean,
+  debug: boolean,
 }
 const props = defineProps<Props>()
 
 const resultsProgs: ProgramData[] = programs.filterPrograms(props.tracksResults)
 
+const resultsProgsReFiltered = computed(() => {
+  // console.log('\nTeeResults > resultsProgsReFiltered...' )
+  // console.log('TeeResults > resultsProgsReFiltered > resultsProgs :', resultsProgs )
+  const results = toRaw(resultsProgs).filter((prog: ProgramData) => {
```

Comment:
question: I'm not quite sure if the `toRaw(...)` function is useful at this point. I tried to use it to extract raw objects from the `proxy` object, but given I later use `JSON.parse(JSON.stringify(....))` it's possible we don't need `toRaw` at all...

Reply:
note: it seems to work equally without calling `toRaw`